app/models/user.rb

```diff
@@ -1193,7 +1196,7 @@ def to_wcif(competition, registration = nil, registrant_id = nil, authorized: fa
       },
       "roles" => roles,
       "assignments" => registration&.assignments&.map(&:to_wcif) || [],
-      "personalBests" => person_pb.map(&:to_wcif),
+      "personalBests" => personal_records,
```

Comment:
Map to WCIF here as per my remark above.

Reply:
Changed